db/upgrade.php

```diff
@@ -66,5 +66,66 @@ function xmldb_block_evasys_sync_upgrade($oldversion) {
         upgrade_block_savepoint(true, 2017121403, 'evasys_sync');
     }
 
+    if ($oldversion < 2019032600) {
+
+        // Define table block_evasys_sync_categories to be created.
+        $table = new xmldb_table('block_evasys_sync_categories');
+        $table->add_field('category_mode', XMLDB_TYPE_INTEGER, '10', null, XMLDB_NOTNULL, null, null);
```

Comment:
You need to specify a default value here or make the column nullalbe:

Debug info: Field block_evasys_sync_categories->category_mode cannot be added. Not null fields added to non empty tables require default value. Create skipped
Error code: ddlunknownerror